changelog/unreleased/pull-2906

```diff
@@ -0,0 +1,9 @@
+Bugfix: Fix inconsistent timestamps and permissions on directories restored
```

Comment:
I'd suggest something like `Restore timestamps and permissions on intermediate directories`. However, I'm not really sure whether `intermediate` is the correct word here: I want to refer to the directories which have to be created to restore a file located in a subdirectory.

Reply:
done